Tests/SwiftLintFrameworkTests/OpeningBraceRuleTests.swift

```diff
@@ -13,9 +13,6 @@ class OpeningBraceRuleTests: SwiftLintTestCase {
             Example("if let a = b { }"),
             Example("while a == b { }"),
             Example("guard let a = b else { }"),
-            Example("if\n\tlet a = b,\n\tlet c = d\n\twhere a == c\n{ }"),
-            Example("while\n\tlet a = b,\n\tlet c = d\n\twhere a == c\n{ }"),
-            Example("guard\n\tlet a = b,\n\tlet c = d\n\twhere a == c else\n{ }"),
```

Comment:
Writing a `where` clause to join multiple conditions is no longer permitted in Swift. Therefore, I have removed this test case.